notebooks/en/labelling_feedback_setfit.ipynb

```diff
@@ -0,0 +1,2874 @@
+{
```

Comment:
<ul><li>The cookbook contains many AI-related topics and it may not be clear from the title that the recipe is talking about data annotation.</li><li>I would suggest to add a person (or multiple people) in "authored by" instead of a company.</li><li>typo: you will learn how to train zero-shot sentiment and topic <strong>classifiers</strong> using SetFit and then use <strong>it</strong> to suggest. Looks like it should be "them", not "it" because you have multiple classifiers.</li><li>I think we can remove the note. The "Open in Colab" option is present for all notebooks, and it would make sense to add it to the "index page", not to an individual notebook. As for "View Source" - this isn't there in the published version. In any case, the instructions that apply to all notebooks, should not be in an individual notebook.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>